packages/tracing-internal/src/browser/browserTracingIntegration.ts

```diff
@@ -364,6 +364,8 @@ export const browserTracingIntegration = ((_options: Partial<BrowserTracingOptio
 export function startBrowserTracingPageLoadSpan(client: Client, spanOptions: StartSpanOptions): Span | undefined {
   client.emit('startPageLoadSpan', spanOptions);
 
+  getCurrentScope().setTransactionName(spanOptions.name);
```

Comment:
I wonder now, if that should possibly go on the isolation scope 🤔 (generally for all our auto-instrumentation). Has up-and downsides... probably it's fine this way, just thinking out loud!

Reply:
yeah, I think in the browser it doesn't matter that much, given we're not automatically forking scopes (afaik) 🤔 

I think on the server, the isolation scope makes a lot of sense since it generally lasts as long as a request, no?